x-pack/plugins/apm/common/rules/apm_rule_types.ts

```diff
@@ -179,24 +180,27 @@ export function formatTransactionErrorRateReason({
 export function formatAnomalyReason({
   serviceName,
   severityLevel,
-  measured,
+  anomalyScore,
   windowSize,
   windowUnit,
+  anomalyDetectorTypes,
 }: {
   serviceName: string;
   severityLevel: string;
-  measured: number;
+  anomalyScore: number;
   windowSize: number;
   windowUnit: string;
+  anomalyDetectorTypes: ApmMlDetectorType[];
 }) {
   return i18n.translate(
     'xpack.apm.alertTypes.transactionDurationAnomaly.reason',
     {
-      defaultMessage: `{severityLevel} anomaly with a score of {measured} was detected in the last {interval} for {serviceName}.`,
+      defaultMessage: `{severityLevel} anomaly with a score of {anomalyScore} and detector type {detectorTypes}, was detected in the last {interval} for {serviceName}.`,
       values: {
         serviceName,
         severityLevel,
-        measured,
+        detectorTypes: anomalyDetectorTypes.join(', '),
```

Comment:
This doesn't seem correct. The `detectorTypes` should be singular here as we group the alerts by the detector index thus the alert will be sent per index.

Reply:
fixed here https://github.com/elastic/kibana/pull/171901/commits/39de1312c108d91c03dfd530f0adb4423d80f52f